_packages/types/src/zods/Profile.ts

```
@@ -0,0 +1,47 @@
+        .array(
+          z.object({
+            title: z.string().optional(),
+            description: z.string().optional(),
+          })
+        )
+        .optional(),
+    })
+    .optional(),
+  social: z
+    .object({
+      website: z.string().url().regex(new RegExp("^https://.")).optional(),
```

Biome (1.9.4)

[error] Use a regular expression literal instead of the RegExp constructor.

Regular expression literals avoid some escaping required in a string literal, and are easier to analyze statically.
Safe fix: Use a literal notation instead.


(lint/complexity/useRegexLiterals)